git_machete/client.py

```diff
@@ -2110,7 +2110,7 @@ def create_github_pr(
         description: str = utils.slurp_file_or_empty(description_path)
 
         ok_str = ' <green><b>OK</b></green>'
-        print(fmt(f'Creating a {"draft " if opt_draft else ""}PR from `{head}` to `{base}`...'), end='', flush=True)
+        print(fmt(f'Creating a {"draft " if opt_draft else ""}PR from `{head}` to `{base}`...'), flush=True)
```

Comment:
Nit: since the string to print is terminated by newline, I suppose flushing happens by default? so `flush=true` is probably redundant